docs/node-operators/get-started/setup-validator.md

```diff
@@ -232,6 +232,29 @@ When a Vega validator node is watching for Ethereum events it will call the `eth
 ```
 
 
+:::warning
+You must select a big enough value for the `MaxEthereumBlocks` because Arbitrum produces about five blocks per second. If the range is too small, your node will not keep up with the rest of the network.
+:::
+
+:::note Find blocks spam for your RPC provider
+You must check how big the span of the blocks you can use with your provider. Otherwise, your validator won't be able to validate events from the Arbitrum network.
```

Comment:
```suggestion
You must check how big the span of the blocks is that you can use with your provider. Otherwise, your validator won't be able to validate events from the Arbitrum network.
```